boards/anavi/anavi-arrows/arrows.py

```diff
@@ -0,0 +1,26 @@
+'''
+KMK keyboard for ANAVI Arrows
+
+This is a macro pad with 4 mechanical switches based on the RP2040. Each key is
+attached to a single GPIO, so the KMK matrix scanner needs to be overridden.
+'''
+
+import board
+
+from kmk.kmk_keyboard import KMKKeyboard
+from kmk.scanners.keypad import KeysScanner
+
+# fmt: off
+_KEY_CFG = [
+    board.D1, board.D2, board.D3, board.D6,
+]
+# fmt: on
+
+
+class AnaviArrows(KMKKeyboard):
+    '''
+    Default keyboard config for ANAVI Arrows.
+    '''
+
+    def __init__(self):
+        self.matrix = KeysScanner(_KEY_CFG)
```

Comment:
Since this is hardware specific and a very short list of pins, there's no need to occupy memory in the global name space.
```suggestion


class AnaviArrows(KMKKeyboard):
    '''
    Default keyboard config for ANAVI Arrows.
    '''

    def __init__(self):
        self.matrix = KeysScanner([board.D1, board.D2, board.D3, board.D6])
```